test/test_torch.py

```diff
@@ -10738,10 +10745,14 @@ def rand_tensor(size, dtype, device):
                 alpha = 0.1
             else:
                 alpha = 3
-            actual = torch.addcmul(a, alpha, b, c)
+            actual = torch.addcmul(a, b, c, value=alpha)
             expected = a + alpha * b * c
             self.assertTrue(torch.allclose(expected, actual))
 
+            self.assertWarnsRegex(
+                lambda: self.assertEqual(actual, torch.addcmul(a, alpha, b, c)),
+                "addcmul is deprecated")
```

Comment:
nit: This is a kind of alarming subset of the string to pattern match on, since it suggests that addcmul in its entirety is deprecated, when in fact it's the specific overload that's deprecated